src/wp-admin/includes/media.php

```diff
@@ -3363,7 +3363,11 @@ function attachment_submitbox_metadata() {
 	if ( isset( $meta['filesize'] ) ) {
 		$file_size = $meta['filesize'];
 	} elseif ( file_exists( $file ) ) {
-		$file_size = filesize( $file );
+		$meta = wp_generate_attachment_metadata( $attachment_id, $file );
```

Comment:
On second thoughts, I think this is a bad idea. Calling this function, generate all image sizes. It would mean on this screen, it maybe slowed down by doing this on all requests.

Reply:
Right, `wp_generate_attachment_metadata()` should not be used here.